Source/Scene/Scene.js

```diff
@@ -1668,7 +1668,7 @@ define([
             // TODO : Execute only opaque and translucent commands for now.
             var startPass = Pass.OPAQUE;
             var endPass = Pass.TRANSLUCENT;
-            for (var pass = startPass; pass < endPass; ++pass) {
+            for (var pass = startPass; pass <= endPass; ++pass) {
```

Comment:
Yup. Most of `executeShadowMapCommands` is temporary just to see things working.